README.md

```diff
@@ -23,9 +23,9 @@
   </a>
 </p>
 
-**semantic-release** automates the whole package release workflow including: determining the next version number, generating the release notes and publishing the package.
+**semantic-release** automates the whole package release workflow including: determining the next version number, generating the release notes, and publishing the package.
```

Comment:
i'm pro oxford comma. happy to discuss if there isnt agreement

Reply:
lol let's not, discuss :D